src/modules/fancyzones/editor/FancyZonesEditor/ViewModels/MonitorViewModel.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Windows;
+using FancyZonesEditor.Utils;
+
+namespace FancyZonesEditor.ViewModels
+{
+    public class MonitorViewModel : INotifyPropertyChanged
+    {
+        private const int MaxPreviewDisplaySize = 160;
+        private const int MinPreviewDisplaySize = 98;
+
+        public event PropertyChangedEventHandler PropertyChanged;
+
+        public delegate void MonitorChangedEventHandler(MonitorChangedEventArgs args);
+
+        public ObservableCollection<MonitorInfoModel> Monitors { get; set; }
```

Comment:
Let's change `Monitors` to a different name (something like `MonitorInfoForViewModel`) because I find it confusing since we also have `App.Overlay.Monitors`.
Ideally it would be better to have only one list of monitors, but let's at least call them differently.